src/js/site.js

```diff
@@ -9,6 +18,17 @@
     });
   });
 
+  [
+    ...all(".folded-summary"),
+    ...all(".folded-details > div .word"),
```

Comment:
Just so I understand; this is for when its expanded right?

I think it's good for now and we can always improve it over time.

One (non-blocking) question: will this clash if there's an anchor link in the summary (there might be for whatever reason, but most likely because code was inlined there with a reference to a term or something like that)?

Reply:
This is for when it's expanded! I will attempt to test the scenario you mentioned.